packages/front-end/pages/settings/index.tsx

```diff
@@ -20,6 +21,9 @@ import { DocLink } from "@/components/DocLink";
 import { useOrganizationMetricDefaults } from "@/hooks/useOrganizationMetricDefaults";
 import { useUser } from "@/services/UserContext";
 import usePermissions from "@/hooks/usePermissions";
+import { GBPremiumBadge } from "@/components/Icons";
+import UpgradeModal from "@/components/Settings/UpgradeModal";
+import EditLicenseForm from "@/components/Settings/EditLicenseForm";
```

Comment:
Nitpick: since this is a modal it would be helpful to have its name reflect that (`EditLicenseModal`)

Reply:
I was following the (mis)convention of EditOrganizationForm (which is also a Modal). But I agree, we can refactor these.